src/bin/init/init.ts

```
@@ -0,0 +1,354 @@
+        buildContext
+    });
+
+    if (hasBeenHandled) {
+        return;
+    }
+
+    await setupEslint({ projectDirPath });
+
+    let doAddRunDevScript = false;
+
+    setup_src: {
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)